code/modules/flufftext/Hallucination.dm

```diff
@@ -159,6 +159,8 @@ Gunshots/explosions/opening doors/less rare audio (done)
 		next_expand = world.time + FAKE_FLOOD_EXPAND_TIME
 
 /obj/effect/hallucination/fake_flood/proc/Expand()
+	if(!target)
+		qdel(src)
```

Comment:
I just noticed I incorrectly fixed this bug a while ago. On line 150 you can see my attempt. The issue comes because the qdel at line 155 does not return.
qdel will not terminate the proc it got called from but will call Destroy. This will null target. Thus causing the bug.
A simple return after line 155 will do. Line 150 and 151 can then also be removed I recon